app/code/Magento/Catalog/Ui/DataProvider/Product/ProductRenderCollectorComposite.php

```diff
@@ -3,6 +3,9 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
+declare(strict_types=1);
```

Comment:
There is no need to add strict types in random files, I believe they should be added in bunches of files automatically at some point.

Reply:
I agree with you.   That was seemingly random on my part.